### DISCUSSION TITLE
Node 8.8 compatible version

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#365] [Q & A] [answered]
Hello. Will there be an ogmios 6+ release for the Node 8.8 version?
Or does is existing one (6.0.3) already compatible?
Thank you!

**Comment by KtorZ** [comment, marked as answer]
>  Will there be an ogmios 6+ release for the Node 8.8 version?

Yes, eventually.

>  Or does is existing one (6.0.3) already compatible?

I can't answer that, there's no 8.8 version so far?

**Reply by mirceahasegan** [reply]
The node 8.8-pre version is out.
I tried starting ogmios 6.0.3 but there are errors such as the one below.
I guess this means it's not compatible with node 8.8.0-pre, correct 😄 ?
```
ogmios-1        | ogmios: DecoderFailure (ChainSync (HardForkBlock (': * ByronBlock (': * (ShelleyBlock (TPraos StandardCrypto) (ShelleyEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (AllegraEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (MaryEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (AlonzoEra StandardCrypto)) (': * (ShelleyBlock (Praos StandardCrypto) (BabbageEra StandardCrypto)) (': * (ShelleyBlock (Praos StandardCrypto) (ConwayEra StandardCrypto)) ('[] *))))))))) (Tip HardForkBlock (': * ByronBlock (': * (ShelleyBlock (TPraos StandardCrypto) (ShelleyEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (AllegraEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (MaryEra StandardCrypto)) (': * (ShelleyBlock (TPraos StandardCrypto) (AlonzoEra StandardCrypto)) (': * (ShelleyBlock (Praos StandardCrypto) (BabbageEra StandardCrypto)) (': * (ShelleyBlock (Praos StandardCrypto) (ConwayEra StandardCrypto)) ('[] *)))))))))) ServerAgency TokNext TokMustReply) (DeserialiseFailure 1828 "expected list len or indef")
```

**Reply by KtorZ** [reply]
Yes. And it's getting slightly exhausting that the ledger breaks its interface on every single release nowadays. It used to be a lot more stable. 

If it's a block in Conway then okay, if it's a block in an earlier era please fill a bug report on the Cardano-ledger repository. It isn't normal.

**Reply by mirceahasegan** [reply]
This occurs on the `sanchonet` testnet. As far as I know, it starts directly in the Conway era, so it is safe to assume it's a conway block.
